<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="7" passed="1" failed="0" skipped="4">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-18T19:50:01 IST" name="Suite" finished-at="2024-09-18T19:50:09 IST" duration-ms="8282">
    <groups>
      <group name="login">
        <method signature="LoginTest.validateUserid()[pri:1, instance:testcases.LoginTest@149e0f5d]" name="validateUserid" class="testcases.LoginTest"/>
        <method signature="LoginTest.validatePassword()[pri:2, instance:testcases.LoginTest@149e0f5d]" name="validatePassword" class="testcases.LoginTest"/>
      </group> <!-- login -->
    </groups>
    <test started-at="2024-09-18T19:50:01 IST" name="LoginTest" finished-at="2024-09-18T19:50:01 IST" duration-ms="68">
      <class name="testcases.ProductSearchTest">
        <test-method signature="productsearch(java.lang.String)[pri:0, instance:testcases.ProductSearchTest@6ef888f6]" started-at="2024-09-18T19:50:01 IST" name="productsearch" depends-on-groups="login" finished-at="2024-09-18T19:50:01 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method ProductSearchTest.productsearch(java.lang.String)[pri:0, instance:testcases.ProductSearchTest@6ef888f6] depends on not successfully finished methods in group "login"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method ProductSearchTest.productsearch(java.lang.String)[pri:0, instance:testcases.ProductSearchTest@6ef888f6] depends on not successfully finished methods in group "login"
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:149)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- productsearch -->
        <test-method signature="addtokart()[pri:0, instance:testcases.ProductSearchTest@6ef888f6]" started-at="2024-09-18T19:50:01 IST" name="addtokart" finished-at="2024-09-18T19:50:01 IST" depends-on-methods="testcases.ProductSearchTest.productsearch" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method testcases.ProductSearchTest.addtokart() on instance testcases.ProductSearchTest@6ef888f6 depends on not successfully finished methods 
[testcases.ProductSearchTest.productsearch() on instance testcases.ProductSearchTest@6ef888f6]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method testcases.ProductSearchTest.addtokart() on instance testcases.ProductSearchTest@6ef888f6 depends on not successfully finished methods 
[testcases.ProductSearchTest.productsearch() on instance testcases.ProductSearchTest@6ef888f6]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:149)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addtokart -->
      </class> <!-- testcases.ProductSearchTest -->
      <class name="testcases.LoginTest">
        <test-method is-config="true" signature="beforeTest(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testcases.LoginTest@149e0f5d]" started-at="2024-09-18T19:50:01 IST" name="beforeTest" finished-at="2024-09-18T19:50:01 IST" duration-ms="20" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[https://www.amazon.in]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[AmazonLogin]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Login]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[false]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[skip test Login]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: skip test Login
at core.Page.beforeTest(Page.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:65)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:381)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:319)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:645)
at org.testng.TestRunner.beforeRun(TestRunner.java:634)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method signature="validateUserid()[pri:1, instance:testcases.LoginTest@149e0f5d]" started-at="2024-09-18T19:50:01 IST" name="validateUserid" finished-at="2024-09-18T19:50:01 IST" duration-ms="0" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[skip test Login]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: skip test Login
at core.Page.beforeTest(Page.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:65)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:381)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:319)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:645)
at org.testng.TestRunner.beforeRun(TestRunner.java:634)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateUserid -->
        <test-method signature="validatePassword()[pri:2, instance:testcases.LoginTest@149e0f5d]" started-at="2024-09-18T19:50:01 IST" name="validatePassword" finished-at="2024-09-18T19:50:01 IST" depends-on-methods="testcases.LoginTest.validateUserid" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method testcases.LoginTest.validatePassword() on instance testcases.LoginTest@149e0f5d depends on not successfully finished methods 
[testcases.LoginTest.validateUserid() on instance testcases.LoginTest@149e0f5d]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method testcases.LoginTest.validatePassword() on instance testcases.LoginTest@149e0f5d depends on not successfully finished methods 
[testcases.LoginTest.validateUserid() on instance testcases.LoginTest@149e0f5d]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:149)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validatePassword -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:testcases.LoginTest@149e0f5d]" started-at="2024-09-18T19:50:01 IST" name="afterTest" finished-at="2024-09-18T19:50:01 IST" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- testcases.LoginTest -->
    </test> <!-- LoginTest -->
    <test started-at="2024-09-18T19:50:01 IST" name="forget" finished-at="2024-09-18T19:50:09 IST" duration-ms="8164">
      <class name="testcases.ForgetTest">
        <test-method is-config="true" signature="beforeTest(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testcases.ForgetTest@1ddf84b8]" started-at="2024-09-18T19:50:01 IST" name="beforeTest" finished-at="2024-09-18T19:50:05 IST" duration-ms="4217" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[https://www.amazon.in]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Forget]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Forget]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method signature="forget()[pri:0, instance:testcases.ForgetTest@1ddf84b8]" started-at="2024-09-18T19:50:05 IST" name="forget" finished-at="2024-09-18T19:50:09 IST" duration-ms="3222" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- forget -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:testcases.ForgetTest@1ddf84b8]" started-at="2024-09-18T19:50:09 IST" name="afterTest" finished-at="2024-09-18T19:50:09 IST" duration-ms="720" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- testcases.ForgetTest -->
    </test> <!-- forget -->
  </suite> <!-- Suite -->
</testng-results>
